docs/frameworks/fastifyjs.md

```diff
@@ -0,0 +1,356 @@
+---
+description: Learn how to deploy a Fastify.js application with Genezio.
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Fastify.js
```

Comment:
From their landing page[1] it doesn't seem that `Fastify` has the suffix `.js` .

I would remove .`js` or `js` terminations from this page to align with their branding.

https://fastify.dev/